data/game_teams_sample.csv

```diff
@@ -0,0 +1,107 @@
+2015020727,4,home,LOSS,REG,Dave Hakstol,2,9,28,8,4,1,57.8,13,3
+2015020643,6,away,LOSS,REG,Claude Julien,2,8,28,11,2,1,50,8,4
+2015020643,4,home,WIN,REG,Dave Hakstol,3,5,25,9,3,0,50,13,4
+2016020747,6,away,LOSS,REG,Claude Julien,2,8,20,13,3,1,48.6,10,5
+2016020747,14,home,WIN,REG,Jon Cooper,3,5,27,21,4,0,51.4,9,2
+2017030211,6,away,WIN,REG,Bruce Cassidy,4,6,27,6,1,1,49,10,8
+2017030211,14,home,LOSS,REG,Jon Cooper,2,9,33,2,3,1,51,7,9
+2013020667,6,away,TIE,REG,Claude Julien,2,5,34,17,3,0,44.4,8,6
+2013020667,26,home,TIE,REG,Darryl Sutter,2,8,27,11,6,1,55.6,11,7
+2014020371,6,away,LOSS,REG,Claude Julien,0,7,36,8,0,0,50,9,0
+2014020371,26,home,WIN,REG,Darryl Sutter,2,8,28,2,3,0,50,14,4
+2016020116,14,away,LOSS,REG,Jon Cooper,1,8,24,7,4,0,45.8,3,8
+2016020116,1,home,WIN,REG,John Hynes,3,5,22,13,1,0,54.2,10,10
+2017030114,14,away,WIN,REG,Jon Cooper,3,9,19,14,5,0,40.9,5,10
+2017030114,1,home,LOSS,REG,John Hynes,1,7,25,12,6,1,59.1,11,12
+2016020592,14,away,TIE,REG,Jon Cooper,2,6,29,6,2,0,42.6,12,3
+2016020592,4,home,TIE,REG,Dave Hakstol,2,11,24,6,2,0,57.4,6,8
+2012020133,14,away,LOSS,REG,Guy Boucher,1,5,17,16,4,0,54.4,9,3
+2012020133,4,home,WIN,REG,Peter Laviolette,2,6,20,20,2,0,45.6,9,5
+2017030213,14,away,WIN,REG,Jon Cooper,2,9,26,19,5,1,54.4,17,11
+2017030213,6,home,LOSS,REG,Bruce Cassidy,1,7,36,27,1,1,45.6,14,5
+2017030214,14,away,LOSS,OT,Jon Cooper,2,7,33,8,4,1,37.9,15,8
+2017030214,6,home,WIN,OT,Bruce Cassidy,3,8,30,8,4,2,62.1,13,12
+2014020845,14,away,LOSS,REG,Jon Cooper,2,5,25,6,2,0,40,11,1
+2014020845,26,home,WIN,REG,Darryl Sutter,3,7,38,4,3,0,60,12,2
+2013020321,14,away,LOSS,REG,Jon Cooper,2,5,22,10,4,1,37.5,15,1
+2013020321,26,home,WIN,REG,Darryl Sutter,3,7,37,8,5,1,62.5,3,4
+2013020285,26,away,WIN,REG,Darryl Sutter,2,5,34,6,3,1,50,4,3
+2013020285,1,home,LOSS,REG,Peter DeBoer,0,6,17,6,2,0,50,14,9
+2014021083,26,away,WIN,REG,Darryl Sutter,3,8,19,4,3,1,51.9,5,8
+2014021083,1,home,LOSS,REG,Adam Oates,1,5,16,8,1,0,48.1,8,4
+2017020156,26,away,WIN,OT,John Stevens,2,7,35,6,2,0,48.6,7,11
+2017020156,6,home,LOSS,OT,Bruce Cassidy,1,7,34,6,2,0,51.4,4,12
+2013020739,26,away,LOSS,REG,Darryl Sutter,2,5,22,15,4,1,63.2,3,3
+2013020739,6,home,WIN,REG,Claude Julien,3,5,29,15,4,1,36.8,8,6
+2014020774,26,away,TIE,REG,Darryl Sutter,2,7,27,10,2,0,42.9,3,8
+2014020774,14,home,TIE,REG,Jon Cooper,2,7,22,10,2,0,57.1,6,6
+2013020088,26,away,LOSS,REG,Darryl Sutter,1,7,24,6,5,0,51.9,8,2
+2013020088,14,home,WIN,REG,Jon Cooper,3,6,24,10,3,0,48.1,7,6
+2016020797,26,away,LOSS,REG,Darryl Sutter,0,7,31,8,2,0,55.4,3,3
+2016020797,14,home,WIN,REG,Jon Cooper,3,5,26,8,2,0,44.6,11,5
+2017020467,26,away,LOSS,REG,John Stevens,1,4,47,6,3,0,43.1,4,8
+2017020467,1,home,WIN,REG,John Hynes,3,6,34,8,2,1,56.9,8,11
+2015020836,26,away,LOSS,REG,Darryl Sutter,0,7,30,6,1,0,55.4,4,4
+2015020836,1,home,WIN,REG,John Hynes,1,4,19,2,3,1,44.6,3,2
+2016020715,26,away,WIN,REG,Darryl Sutter,3,6,13,8,2,1,51.1,1,5
+2016020715,1,home,LOSS,REG,John Hynes,1,6,16,4,4,0,48.9,14,2
```

Comment:
Nice! I like having more values here. It's great for if we need to expand our calculations because new data (or the full data, when it comes to the spec) messes it up.